app/src/main/java/com/glowstudio/android/blindsjn/feature/paymanagement/PayManagementScreen.kt

```
@@ -0,0 +1,506 @@
+                                }
+                                Row(
+                                    modifier = Modifier.fillMaxSize(),
+                                    verticalAlignment = Alignment.Bottom,
+                                    horizontalArrangement = Arrangement.spacedBy(4.dp, Alignment.CenterHorizontally)
+                                ) {
+                                    barData.forEachIndexed { idx, value ->
+                                        val isToday = idx == barData.lastIndex
+                                        val isDanger = marginRates[idx] < 20
+                                        Column(horizontalAlignment = Alignment.CenterHorizontally) {
+                                            // 금액 라벨 (compact)
+                                            val compactValue = if (value >= 10000) String.format("%.1f만", value / 10000f) else value.toString()
```

detekt (1.23.8)

[warning] String.format("%.1f만", value / 10000f) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)